### DISCUSSION TITLE
Hunt API?

### DISCUSSION THREAD

**Discussion by MobileAppSec** [Security-Onion-Solutions/securityonion#12426] [Unsupported Versions]
We have analysts manually inputting pre built hunt queues either through the drop downs or query field. Looking to automate this task. Is there a hunt API or other API that would help with this automation? Much thanks to the community...

**Comment by rahatUlAin** [comment]
Find here https://docs.securityonion.net/en/2.4/soc-customization.html#custom-queries)

**Reply by MobileAppSec** [reply]
Perhaps I need to explain the use case a little better as I'm not sure how that link answers my question. I already have a long list of pre built queries in an Excel file. I want to use an API to iterate through the list, running reach query and then doing "stuff" with the output.
